src/Sylius/Bundle/PromotionBundle/Form/Type/Action/FixedDiscountConfigurationType.php

```diff
@@ -27,10 +27,6 @@ public function buildForm(FormBuilderInterface $builder, array $options): void
         $builder
             ->add('amount', MoneyType::class, [
                 'label' => 'sylius.form.promotion_action.fixed_discount_configuration.amount',
-                'constraints' => [
```

Comment:
If somebody made use of `FixedDiscountConfigurationType` the same way `ChannelBasedFixedDiscountConfigurationType` using it, but in their custom promotion action, then validation rules will disappear for them, so I think this is a BC.

There at least should be a mention in UPGRADE. Same for other PromotionBundle form types extended in core.

https://github.com/Sylius/Sylius/blob/8872bafa942392ca93cbd9d99fef14ede79439a4/src/Sylius/Bundle/CoreBundle/Form/Type/Promotion/Action/ChannelBasedFixedDiscountConfigurationType.php#L26-L33

Reply:
After discussing with the team, we have determined that this is not a BC, but it is important to add information about this to the upgrade file. I have added it.